### ISSUE TITLE
REQ: Help listening to timed metadata from FlasHLS

### ISSUE THREAD

**Issue by jstnkrr** [clappr/clappr#1285]
Hi 

First of all thanks to all for their great work on this project.

We'd like to be able to figure out how to bind a listener to FlasHLS's **HLSEvent.ID3_UPDATED** event [(documentation here)](https://github.com/mangui/flashls/blob/dev/API.md).

[PLAYBACK_LOADEDMETADATA](http://clappr.github.io/classes/Playback.html#event_PLAYBACK_LOADEDMETADATA) did not fire in our tests.

```
this.clappr.listenTo(
			this.clappr.core.getCurrentPlayback(),
			Clappr.Events.PLAYBACK_LOADEDMETADATA,
			function(event) {
				console.log("PLAYBACK_LOADEDMETADATA ", event);

			}
		);
```

Can anyone please point us in a helpful direction? We've managed to add listeners on html_video and hls.js playback engines, but not FlasHLS.

**Comment by jstnkrr** [comment]
I'm sorry: PLAYBACK_LOADEDMETADATA **does fire**, but does not contain any ID3 metadata. And it only fires when loading the first fragment.

**Comment by jstnkrr** [comment]
Actually I may have solved this on my own.

Clappr's HLSPlayer.swf seems to have a similar events API to the [Chromless player](http://www.flashls.org/latest/examples/chromeless/).  

I added a new emitter in `playbacks/flashls/flashls_events.js`:

```
 id3Updated(id3Data) {
    Mediator.trigger(`${this.instanceId}:id3updated`, id3Data)
 }
```

I also added listeners in `flashls.js` to emit a corresponding `playback:id3updated` event.  I can now listen to the event in my app.

I'm not sure if it's better to use the PLAYBACK_LOADEDMETADATA event, so for now I created a separate one.  Would a new event like PLAYBACK_UPDATEDMETADATA work better?

For now I'll close this issue. I'm happy to discuss further if anyone is interested.
